### DISCUSSION TITLE
How to modify the "path" and "addr" parameter in config file?

### DISCUSSION THREAD

**Discussion by tcexeexe** [open-compass/opencompass#159] [Q&A] [answered]
Hi~
     I tried to test a new model through the API interface.l have modify the "configs/eval_gpt3.5.py" and "configs/models/gpt_3.5_turbo.py" . It works. 
However, How to modify the "path" and "addr" parameter in following code?
```
models = [
    dict(abbr='GPT-3.5-turbo-0613',
        type=OpenAI, path='gpt-3.5-turbo-0613',
        key='ENV',  # The key will be obtained from $OPENAI_API_KEY, but you can write down your key here as well
        meta_template=api_meta_template,
        query_per_second=1,
        max_out_len=2048, max_seq_len=2048, batch_size=8),
]
```
Now l use path='gpt-3.5-turbo-0613' abbr='GPT-3.5-turbo-0613. No errors were reported .
Are there any restrictions on these two parameters, or are they arbitrary? Can you give me some examples?

**Comment by Leymore** [comment, marked as answer]
In the case of `OpenAI`, `path` is which openai api model to use, possible values are listed in openai [doc](https://platform.openai.com/docs/models/gpt-3-5). While `abbr` is an arbitrary abbreviation you make to name this model, there is no restriction.

You can read the comment in source code for more infomation https://github.com/InternLM/opencompass/blob/bbdedc6c95271791a6e5429bc78fc33bc6eaae53/opencompass/models/openai_api.py#L20-L27

**Reply by tcexeexe** [reply]
got it,thanks
